addons/udes_stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -877,6 +889,16 @@ def _create_batch(self, user_id, picking_type_id, picking_priorities=None, picki
         The batch will be marked as ephemeral.
         In case no pickings exist, return None.
         """
+        # TODO - This function actually searches for one picking but it talks about pickings... why?
+        #        What is this function actually supposed to do?
```

Comment:
Yeah it looks like single picking. I guess the only way for multiple if you pass multiple picking_id's into picking_id? Looks like it might be realted to matrix picking for some stuff when glancing at 11

Reply:
I guess nothing prevents to have an overridden version of `search_for_pickings` where default limit value is >1. And I guess the tests were written taking into account that the default is 1 in this module. When `u_reserve_pallet_per_picking` = True, more pickings can be added using `add_extra_pickings`.